app/controllers/network_routes_controller.go

```diff
@@ -1502,6 +1503,14 @@ func NewNetworkRoutingController(clientset *kubernetes.Clientset,
 	nrc.bgpRRServer = false
 	nrc.bgpServerStarted = false
 
+	nrc.cniConfFile = os.Getenv("KUBE_ROUTER_CNI_CONF_FILE")
+	if nrc.cniConfFile == "" {
+		nrc.cniConfFile = "/etc/cni/net.d/10-kuberouter.conf"
+	}
+	if _, err := os.Stat(nrc.cniConfFile); os.IsNotExist(err) {
+		return nil, errors.New("Invalid CNI conf file " + nrc.cniConfFile)
```

Comment:
`s/Invalid CNI conf/CNI conf file does not exist` - the current error sounds like the file exists but it is invalid

Reply:
Will change it.